### PR TITLE
Possible fix for performance issue with accounts GQL query

### PR DESCRIPTION
**Pull request by n1tranquilla**
## Describe your changes
* Instead of loading all internal commands into memory and then truncating, load only the limit
* There are several iterations taken per internal command loaded into memory, so this should have a cascading effect (in a good way)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] I have verified new and existing tests pass locally with my changes.
- [x] I verified whether it was necessary to increment the database version.

### CODE REVIEW

File: rust/src/web/graphql/feetransfers/mod.rs [hunk 1]
```diff
@@ -253,8 +253,8 @@ impl FeetransferQueryRoot {
         // TODO
         // recipient query
         if let Some(recipient) = query.as_ref().and_then(|q| q.recipient.clone()) {
-            let mut fee_transfers: Vec<FeetransferWithMeta> = db
-                .get_internal_commands_public_key(&recipient.into())?
+            let fee_transfers: Vec<FeetransferWithMeta> = db
+                .get_internal_commands_public_key(&recipient.into(), limit as u32)?
```
Comment (jhult) [inline rust/src/web/graphql/feetransfers/mod.rs:257]: Can you limit this early? Don't you need to wait for all other filters to be complete?

Comment (n1tranquilla) [reply rust/src/web/graphql/feetransfers/mod.rs:257]: I'm going to resolve this for now. If there is more improvement that can be made, let's follow up with another PR.

**Review by jhult — APPROVED** [review]
LGTM

**Comment by jhult** [comment]
This may resolve #1339

**Pull request merged**